### DISCUSSION TITLE
1.3.14

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#10302] [General]
*  Update Microsoft.WSL.Kernel to version 5.15.90.3-1
*  Update MSRDC to 1.2.4419
*  Update Microsoft.WSLg to version 1.0.55
*  Fix /run/user/<uid> being hidden when a new login session is created by symli... (solves #10205)
*  Implement a --no-distribution option in wsl --install

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/1.3.14'>1.3.14</a>.</em>

**Comment by samuk190** [comment]
auto reclaim still works on this version ?

**Comment by crramirez** [comment]
About **Implement a --no-distribution option in wsl --install**

It is already implemented in the built-in wsl version. Why is necessary to implement here? To install WSL1? Or you are in a case that the user installed the Store application directly without installing WSL?

Is this change associated to an issue here in the repo?

**Comment by g2flyer** [comment]
Not sure to which of above changes this is related but since upgrading from 1.3.11 to 1.3.14 on my win11 (10.0.22621) a few days ago i get notification "An Http proxy change has been detected on the host. Please restart WSL to apply the chaneg" each time my (host) vpn connects or disconnects. Besides wondering why i should have to restart WSL -- it worked before in such situations and it works since ... -- the more annoying aspect is that on vpn connects (but not on disconnects, or at least not always ..) all my X11-windows disappear, also from taskbar. As soon as i start a new x-app, e.g. an xterm, though, all re-appear but also all move to the current virtual desktop (whereas before i had them spread over several ones).  

Any thoughts?  I can create an issue but thought i mention it here first for some initial sights (e.g., at least if i should create issue for wslg or wsl :-)

**Reply by khusika** [reply]
Not only you :)
See https://github.com/microsoft/WSL/issues/10305#issuecomment-1640023394

**Reply by g2flyer** [reply]
Ah, i looked at title of issues with nothing relevant and searched also issues for proxy, but i guess github search is not smart enough yet to search also text in embedded images :-)  That said, i don't seem to see the original problem with the linked issue. But thanks for the link, will monitor that ...

**Comment by GXC2356** [comment]
Guys can anyone help me plz about xfce4 on wsl
at first I was facing this error ' System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
Failed to talk to init daemon ' (was an issue and I wanted to solve to prevent any problem while building ROM)

After solving it (by downloading wsl app) wen I try to launch xfce 4 it says ' Starting xrdp (via systemctl): xrdp.service ' and doesn't launch in remote desktop, before it was saying ' Starting protocol server ( ok ) ' and was working fine

Can anyone help me, I will be thankful also sry if I breaked any rule I'm new here 😅

**Comment by muratyurdakul75** [comment]
wsl2 still does not support more than 64 processors. There are so many complaints about it. I hope this will be fixed in the first update.

**Reply by muratyurdakul75** [reply]
In some projects, the computer only runs wsl. Therefore, it is important that we can use more processors. My computer has 2 x E5 2698 V4 processor. But unfortunately wsl uses half of it. A computer running wsl with Docker Dosktop. He has no other job. Therefore, it would be very beneficial if we could use more processors.

**Comment by zcobol** [comment]
@muratyurdakul75 if your computer `only runs wsl` why not install Linux on it and take advantage of all the hardware? Running WSL on a Windows host has serious limitations.

**Reply by muratyurdakul75** [reply]
@zcobol  There are 2 applications that monitor the container I use. Unfortunately, these applications can run on Windows. There are no Linux versions. That's why I have to use Windows.
